jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -2579,6 +2581,189 @@ ecma_builtin_array_prototype_object_copy_within (const ecma_value_t args[], /**<
 
   return ecma_copy_value (ecma_make_object_value (obj_p));
 } /* ecma_builtin_array_prototype_object_copy_within */
+
+static uint32_t
+ecma_builtin_array_flatten_into_array (ecma_value_t target,
```

Comment:
The return value is not ideal: in case of any errors it should be returned correctly.

Reply:
The method's comments seems to be misplaced.